src/CribblyBackend.DataAccess/Migrations/20210318091908_AddStandingsData.cs

```diff
@@ -0,0 +1,29 @@
+using FluentMigrator;
+
+namespace CribblyBackend.Migrations
+{
+    [Migration(20210318091908)]
+    public class AddStandingsData : Migration
+    {
+        public override void Down()
+        {
+            Delete.Column("Division").FromTable("Teams");
+            Delete.Column("Wins").FromTable("Teams");
+            Delete.Column("Losses").FromTable("Teams");
+            Delete.Column("TotalScore").FromTable("Teams");
+            Delete.Column("Seed").FromTable("Teams");
+            Delete.Column("InTournament").FromTable("Teams");
+        }
+
+        public override void Up()
+        {
+            Alter.Table("Teams")
+                .AddColumn("Division").AsString()
+                .AddColumn("Wins").AsInt32()
+                .AddColumn("Losses").AsInt32()
+                .AddColumn("TotalScore").AsInt32()
+                .AddColumn("Seed").AsInt32()
+                .AddColumn("InTournament").AsBoolean();
```

Comment:
:crystal_ball: We should think a bit about our data design. Do we need to store the number of wins/losses, or can that be calculated from the `Games` table? Do we want to get fancy and have the wins and losses columns be auto updated in a MySQL trigger when a game gets recorded?

Same goes for total score and `InTournament` - do we think these need to be stored in the DB, or can they be calculated from other data?

Reply:
Is it best practice to have the least amount of data in the DB and calculate things whenever possible?

Reply:
Thinking about this one more, I suppose since we have abstracted data access away, we can have calculations happen at the service level

Reply:
^ yes to having calculations happen at the service level. As for storing the least amount of data, it kind of depends on our goals. Storing more data == fewer calculations; storing less data == more calculations. Just a trade off we need to decide on, I suppose. At the end of the day, though, we aren't going to be scaling massively so either way probably doesn't make much of a difference. I was just thinking it's simpler for us to calculate something on the backend than it is to make sure it stays up to date in the db.

Reply:
So this is sort of resolved. The calculations can now happen at the service level, but it is going to depend on the ability to search for games, or return all games and run a LINQ query (code that is not yet written).